src/core/Cline.ts

```diff
@@ -1479,6 +1485,323 @@ export class Cline {
 							break
 						}
 					}
+
+					case "insert_code_block": {
+						const relPath: string | undefined = block.params.path
+						const operations: string | undefined = block.params.operations
```

Comment:
One small thing - we should add toolParams.operations to the check here: https://github.com/RooVetGit/Roo-Code/blob/3862057a4d7de81bbf317fca067b42247d4a40c2/src/shared/modes.ts#L201